Frontend/CO.CDP.OrganisationApp.Tests/Extensions/StringExtensionsTests.cs

```diff
@@ -0,0 +1,18 @@
+using CO.CDP.OrganisationApp.Extensions;
+using FluentAssertions;
+
+namespace CO.CDP.OrganisationApp.Tests.Extensions;
+
+public class StringExtensionsTests
+{
+    [Theory]
+    [InlineData("validated-text")]
+    [InlineData("  \tvalidated-text \x0c ")]
+    [InlineData("\rv\u200Cal\u200Did\u200Bate\u2060d-\u180Etex\uFEFFt\uFEFF")]
```

Comment:
Can we add a case for repeated characters to make sure it doesn't just remove one?

i.e. `[InlineData("ab\u200Ccd\u200Cef\u200C")]`